clients/ctl/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -137,7 +143,10 @@ export const useDataUse = (): TaxonomyHookData<DataUse> => {
       legal_basis: du.legal_basis,
       special_category: du.special_category,
       recipients: du.recipients ?? [],
-      legitimate_interest: du.legitimate_interest,
+      legitimate_interest:
+        du.legitimate_interest == null
```

Comment:
Should we use triple `===`? 👀

Reply:
yah I think there are different opinions on this. I've always gone with 'always use `===`, except `==` is useful if you want to see if a value is `undefined` **or** `null`'. more info on that in [this article](https://blog.webdevsimplified.com/2020-08/==-vs-===/) (see the "When to use Double Equals" section. the article is very strongly titled considering they don't actually recommend to _totally_ stop using double equals 😅 )

Reply:
in this case it's only comparing to `undefined` though, so I'll change to `=== undefined` 👍

Reply:
oh never mind... it turns out it _can_ be null or undefined, so the `== null` is still more handy here. thanks to the tests which caught that 😅